sdk/textanalytics/azure-ai-textanalytics/samples/sample_analyze.py

```diff
@@ -53,54 +53,55 @@ def analyze(self):
             The only complaint I have is the food didn't come fast enough. Overall I highly recommend it!"
         ]
 
-        poller = text_analytics_client.begin_analyze(
+        poller = text_analytics_client.begin_analyze_batch_actions(
             documents,
             display_name="Sample Text Analysis",
-            entities_recognition_tasks=[EntitiesRecognitionTask()],
-            pii_entities_recognition_tasks=[PiiEntitiesRecognitionTask()],
-            key_phrase_extraction_tasks=[KeyPhraseExtractionTask()]
+            actions=[
+                RecognizeEntitiesAction(),
+                RecognizePiiEntitiesAction(),
+                ExtractKeyPhrasesAction()
+            ],
         )
 
         result = poller.result()
 
-        for page in result:
-            for task in page.entities_recognition_results:
-                print("Results of Entities Recognition task:")
-
-                docs = [doc for doc in task.results if not doc.is_error]
-                for idx, doc in enumerate(docs):
-                    print("\nDocument text: {}".format(documents[idx]))
-                    for entity in doc.entities:
-                        print("Entity: {}".format(entity.text))
-                        print("...Category: {}".format(entity.category))
-                        print("...Confidence Score: {}".format(entity.confidence_score))
-                        print("...Offset: {}".format(entity.offset))
-                    print("------------------------------------------")
-
-            for task in page.pii_entities_recognition_results:
-                print("Results of PII Entities Recognition task:")
-
-                docs = [doc for doc in task.results if not doc.is_error]
-                for idx, doc in enumerate(docs):
-                    print("Document text: {}".format(documents[idx]))
-                    for entity in doc.entities:
-                        print("Entity: {}".format(entity.text))
-                        print("Category: {}".format(entity.category))
-                        print("Confidence Score: {}\n".format(entity.confidence_score))
-                    print("------------------------------------------")
-
-            for task in page.key_phrase_extraction_results:
-                print("Results of Key Phrase Extraction task:")
-
-                docs = [doc for doc in task.results if not doc.is_error]
-                for idx, doc in enumerate(docs):
-                    print("Document text: {}\n".format(documents[idx]))
-                    print("Key Phrases: {}\n".format(doc.key_phrases))
-                    print("------------------------------------------")
+        first_action_result = next(result)
```

Comment:
u are already checking if the document didn't have errors. it will be good to also check that the action didn't have errors